examples/hierarchical_pipeline.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<p>Let's use the pipeline to directly forecast the middle level in the beginning of the chapter and then after each subchapter run backtest on hierarchicals. This way in the end we will have the list of dataframes with metrics and in the end of the chapter we will simply merge them  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1070/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Reply:
<p>This is the same as it is right now, but with a baseline at the beginning. I don't think it is better to start the section with a baseline, because the main point is to show how to work with reconciliation.</p>